src/FileSavers/ProjectDescriptionSaver.cs

```diff
@@ -312,12 +330,14 @@ private static void SaveSharedFile(ParametersForSave par)
         private const int mainTechObjectsFileVersion = 1;
         private const int mainTechDevicesFileVersion = 1;
         private const int mainRestrictionsFileVersion = 1;
+        private const int mainRecipesFileVersion = 1;
         private const int mainPRGFileVersion = 1;
 
         private const string mainIOFileName = "main.io.lua";
         public const string MainTechObjectsFileName = "main.objects.lua";
         private const string mainTechDevicesFileName = "main.devices.lua";
         public const string MainRestrictionsFileName = "main.restrictions.lua";
+        public const string MainRecipesFileName = "recipes.lua";
```

Comment:
```suggestion
        public const string MainRecipesFileName = "main.recipes.lua";
```

Я бы предпочел такое.

Reply:
Ok, все равно сейчас код перебираю, исправлю.